### PR TITLE
h.264 fixes

### PR DESCRIPTION
**Pull request by k0nserv**
Fixes for h.264 discovered by sending from Windows usin Nvidia HVEC.

### CODE REVIEW

File: src/packet/h264.rs [hunk 1]
```diff
@@ -208,7 +208,7 @@ impl Depacketizer for H264Depacketizer {
         out: &mut Vec<u8>,
         _: &mut CodecExtra,
     ) -> Result<(), PacketError> {
-        if packet.len() <= 2 {
+        if packet.len() == 0 {
```
Comment (algesten) [inline src/packet/h264.rs:211]: Why can this be zero?

**Review by algesten — APPROVED** [review]

File: src/packet/h264.rs (hunk 1 above)
Comment (k0nserv) [reply src/packet/h264.rs:211]: It can't ;)

Comment (k0nserv) [reply src/packet/h264.rs:211]: Could write this condition as `packet.len() < 1` if you want

Comment (algesten) [reply src/packet/h264.rs:211]: Right. So what was wrong with <= 2? :)

Comment (k0nserv) [reply src/packet/h264.rs:211]: It errors on payloads of length 1 and 2, which are allowed

Comment (algesten) [reply src/packet/h264.rs:211]: I copied this code from webrtc-rs – they would have the bug too. Maybe we should upstream?

Comment (algesten) [reply src/packet/h264.rs:211]: Oh. I missed that. Thanks.

**Pull request merged**